kubernetes/windows-exporter-daemonset.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: windows-exporter
+  name: windows-exporter
+  namespace: monitoring
+spec:
+  selector:
+    matchLabels:
+      app: windows-exporter
+  template:
+    metadata:
+      labels:
+        app: windows-exporter
+    spec:
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\system"
+      hostNetwork: true
+      initContainers:
+        - name: configure-firewall
+          image: mcr.microsoft.com/windows/nanoserver:1809
+          command: ["powershell"]
+          args: ["New-NetFirewallRule", "-DisplayName", "'windows-exporter'", "-Direction", "inbound", "-Profile", "Any", "-Action", "Allow", "-LocalPort", "9182", "-Protocol", "TCP"]
+      containers:
+      - args: 
+        - --config.file=%CONTAINER_SANDBOX_MOUNT_POINT%/config.yml
+        name: windows-exporter
+        image: jsturtevant/windows-exporter:latest
```

Comment:
@carlpett  I need to update this but currently not pushing a latest tag.  We won't have a tag until the git repository is tagged and released.  Thoughts?

the current format is `ghcr.io/prometheus-community/windows-exporter:<versionnumber>`

Reply:
Based on other unreleased changes, the next version is going to be 0.18.
I'm not sure if we want a `latest` tag? On the other hand, documenting it as a fixed version, I doubt we'll remember to update it...

Reply:
Thinking some more, I think we'll want a "bleeding edge" image. For example, the `node_exporter` has a `master` tag. They also have a `latest`, pointing to latest release. I'm personally not a fan, but I suppose there's probably high demand for it?

Reply:
added a "bleeding" edge image.  let me know if you want me to squash the commits